### ISSUE TITLE
Wrong images generated

### ISSUE THREAD

**Issue by taylorlu** [lelechen63/ATVGnet#15]
Hi, I clone the code and run the demo.py, nothing error, but the video and the images generated are totally in wrong type like this.
<table>
  <tr>
    <th> img</th>
    <th><img src="https://user-images.githubusercontent.com/5125756/62290498-14e19b80-b494-11e9-952f-4f23965e4bc0.png" ></th>
  </tr>
  <tr>
    <th>motion</th>
    <th><img src="https://user-images.githubusercontent.com/5125756/62290928-1790c080-b495-11e9-8c99-942cd9dff2cf.png" ></th>
  </tr>
  <tr>
    <th>attention</th>
    <th><img src="https://user-images.githubusercontent.com/5125756/62290936-1eb7ce80-b495-11e9-90d3-b48b90926408.png" ></th>
  </tr>
</table>

By the way, I run in `CPU` mode.
thanks in advance.

**Comment by lelechen63** [comment]
are you running with pytorch 0.4.1? I haven't tested the model on other versions.

**Comment by taylorlu** [comment]
I'm sure it's under torch 0.4.1.
```
Python 2.7.10 (default, Feb 22 2019, 21:55:15) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.37.14)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> print torch.__version__
0.4.1
```

**Comment by lelechen63** [comment]
Ok. Are you running with the provided model (https://drive.google.com/drive/folders/1WYhqKBFX6mLtdJ8sYVLdWUqp5FJDmphg)? Are you running the demo code with provided images and audios? I just tried and it works fine for me.

**Comment by taylorlu** [comment]
The trained model was downloaded according to tip3, it's all right.
and the images and audios was not modified yet.
```
def parse_args():
    parser = argparse.ArgumentParser()

    parser.add_argument("--batch_size",
                     type=int,
                     default=1)
    parser.add_argument("--cuda",
                     default=False)
    parser.add_argument("--lstm",
                     default=True)
    parser.add_argument("--vg_model",
                     type=str,
                     default="../model/generator_23.pth")
    parser.add_argument("--at_model",
                     type=str,
                     # default="/u/lchen63/lrw/model/audio2lmark_pca/audio2lmark_24.pth")
                     default="../model/atnet_lstm_18.pth")
    parser.add_argument( "--sample_dir",
                    type=str,
                    default="../results")
                    # default='/media/lele/DATA/lrw/data2/sample/lstm_gan')test
    parser.add_argument('-i','--in_file', type=str, default='../audio/test.wav')
    parser.add_argument('-d','--data_path', type=str, default='../basics')
    parser.add_argument('-p','--person', type=str, default='../image/musk1.jpg')
    parser.add_argument('--device_ids', type=str, default='0')
    parser.add_argument('--num_thread', type=int, default=1)   
    return parser.parse_args()
config = parse_args()
```

**Comment by taylorlu** [comment]
And after preprocessing, it generates 3 files: `musk1_region.jpg, musk1_lmark.png, musk1.npy`
![775B1BF4-A83C-49EA-90A3-079E91E153EE](https://user-images.githubusercontent.com/5125756/62343588-d85a8200-b51d-11e9-8c48-4eeeb1eb8e48.png)

**Comment by taylorlu** [comment]
Thanks, I have work it out, it's the reason of `CPU` mode, the pth must run under CUDA.

**Comment by taylorlu** [comment]
I have found that the Normalize method handle abnormally under CPU mode, it generates different shape and value compare with which under GPU.
```
transform = transforms.Compose([
     transforms.ToTensor(),
     transforms.Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5))
 ])
```
The GPU output [3,128,128], while CPU output [128,128,3], and a lot of values also different.

**Comment by lelechen63** [comment]
That is interesting. I will write it in readme since I haven't tested the CPU version. Thanks to letting me know this.
